src/lib/api-docs/types.ts

```diff
@@ -14,5 +14,5 @@ export interface ApiDocsVersionData {
 	// The release stage of this version of the API docs
 	releaseStage?: string // typically 'stable' | 'preview'
 	// The schema file we'll load and render into the page for this version
-	targetFile: GithubFile | string
+	sourceFile: GithubFile | string
```

Comment:
This is the crux of the change, for `OpenApiDocsView`, `sourceFile` here felt like a much better name than `targetFile`, which resulted in the initial divergence. In brief, this PR cleans up the messy divergence and resolves related duplicate type interfaces.